projects/step-frontend/src/lib/os-plugins/modules/function-packages/function-packages.module.ts

```diff
@@ -74,7 +74,7 @@ export class FunctionPackagesModule {
 
   private registerViews(): void {
     this._viewRegistry.registerRoute({
-      path: 'functionPackages',
+      path: 'function-packages',
```

Comment:
There are places in project, that still references to `functionPackages` link

Reply:
Fixed.